cax-ap-cam/README.md

```diff
@@ -1,2 +1,2 @@
-# li-ap-energy
+# cax-ap-cam
```

Comment:
I suggest bl-ap-imgproc for package and IOC name

Reply:
done.